docs/docs/customization/themes.md

```diff
@@ -0,0 +1,95 @@
+Pathfora has two basic pre-built color themes, light (default if no theme is set) and dark. Furthermore, the user can set their own custom colors for any element of the module within the javascript configuration. For more advanced color and style customization you can [add your own CSS](/customization/css.md).
+
+## theme
+Set the color scheme of the module, this can be a predefined theme or indicate that the module should have custom colors.
+
+| Value | Type | Behavior |
+|---|---|---|
+| light | string | `default` default light color scheme |
+| dark | string | a predefined darker color scheme |  
+| custom | string | the user should define custom colors using `colors` |  
+
+### Dark Theme - [Live Preview](../../examples/preview/customization/themes/dark.html)
+
+![Form Placeholders](../examples/img/customization/themes/dark.png)
+
+<pre data-src="../../examples/src/customization/themes/dark.js"></pre>
+
+
+## colors
+Define a set of custom colors for each element of the module.
+
+<table>
+  <tr>
+    <th>VALUE</th>
+    <th>TYPE</th>
+    <th>BEHAVIOR</th>
+  </tr>
```

Comment:
This `tr` needs to be in a `thead`. The layout is borked otherwise.

Why not use a markdown table here?